File: ios/CoolWebViewManager.m
//
//  CoolWebViewManager.m
//  FriendsBook
//
//  Created by Victor Valle-Castillo on 2024-06-10.
//

#import "React/RCTViewManager.h"

@interface RCT_EXTERN_MODULE(CoolWebViewManager, RCTViewManager)
  RCT_EXPORT_VIEW_PROPERTY(htmlString, NSString)
  RCT_EXPORT_VIEW_PROPERTY(onComment, RCTDirectoEventBlock)
  RCT_EXPORT_VIEW_PROPERTY(onHighlight, RCTDirectoEventBlock)
@end

File: ios/CoolWebViewManager.swift
//
//  CoolWebViewManager.swift
//  FriendsBook
//
//  Created by Victor Valle-Castillo on 2024-06-10.
//

import Foundation

@objc(CoolWebViewManager)
class CoolWebViewManager: RCTViewManager {
  override func view() -> UIView! {
    return CoolWebView()
  }
  
  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
}
